src/user.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * @fileoverview Genotet user.
+ */
+
+'use strict';
+
+/** @const */
+genotet.user = {};
+
+/**
+ * User information are saved to this URL via http and received via jsonp.
+ * @type {!Object<{
+ *   username: string,
+ *   sessionId: string,
+ *   expiration: number
+ * }>}
+ */
+genotet.user.info;
```

Comment:
Now this is undefined, rather than null. You need to either give it null explicitly, or add undefined to type.